### PR TITLE
Migration to aiida 1.0

### PR DESCRIPTION
**Pull request by yakutovicha**

### CODE REVIEW

**Comment by yakutovicha** [comment]
would @sphuber or @ltalirz volunteer to review?

**Comment by ltalirz** [comment]
thanks for your work!
I'm not sure I'll have time this week as 3 days are already spent at the invenio bootcamp.
if I don't manage to test, we can check it together next week.

File: aiida_cp2k/__init__.py [hunk 1]
```diff
@@ -2,10 +2,10 @@
 ###############################################################################
 # Copyright (c), The AiiDA-CP2K authors.                                      #
 # SPDX-License-Identifier: MIT                                                #
-# AiiDA-CP2K is hosted on GitHub at https://github.com/cp2k/aiida-cp2k        #
+# AiiDA-CP2K is hosted on GitHub at https://github.com/aiidateam/aiida-cp2k   #
 # For further information on the license, see the LICENSE.txt file.           #
 ###############################################################################
-
+"""AiiDA-CP2K plugins, parsers, workflows, etc ..."""
 __version__ = "0.9"
```
Comment (ltalirz) [inline aiida_cp2k/__init__.py]: please bump the MAJOR version

File: aiida_cp2k/calculations/__init__.py [hunk 2]
```diff
@@ -2,230 +2,160 @@
...
+                                '{:<15} {:<15} {:<15}'.format(*self.inputs.structure.cell[i]))
             topo = "FORCE_EVAL/SUBSYS/TOPOLOGY"
-            inp.add_keyword(topo + "/COORD_FILE_NAME", self._COORDS_FILE_NAME)
+            inp.add_keyword(topo + "/COORD_FILE_NAME", self._DEFAULT_COORDS_FILE_NAME)
             inp.add_keyword(topo + "/COORD_FILE_FORMAT", "XYZ")
-        inp_fn = tempfolder.get_abs_path(self._INPUT_FILE_NAME)
-        with open(inp_fn, "w") as f:
-            f.write(inp.render())
+
+        with open(folder.get_abs_path(self._DEFAULT_INPUT_FILE), "w") as fobj:
```
Comment (dev-zero) [inline aiida_cp2k/calculations/__init__.py]: maybe use `io.open` for consistency?

File: aiida_cp2k/calculations/__init__.py [hunk 3]
```diff
@@ -2,230 +2,160 @@
...
 
         # check for left over settings
         if settings:
-            msg = "The following keys have been found "
-            msg += "in the settings input node {}, ".format(self.pk)
-            msg += "but were not understood: " + ",".join(settings.keys())
-            raise InputValidationError(msg)
+            raise InputValidationError("The following keys have been found " +
+                                       "in the settings input node {}, ".format(self.pk) + "but were not understood: " +
+                                       ",".join(list(settings.keys())))
```
Comment (dev-zero) [inline aiida_cp2k/calculations/__init__.py]: `str.join()` works with iterators and views as well, no need to convert to a full list here.

File: aiida_cp2k/calculations/__init__.py [hunk 4]
```diff
@@ -2,230 +2,160 @@
...
+    def add_keyword(self, kwpath, value):
         self._add_keyword_low(kwpath.split("/"), value, self.params)
 
     # --------------------------------------------------------------------------
     def _add_keyword_low(self, kwpath, value, params):
+        """Adds keyword"""
         if len(kwpath) == 1:
             params[kwpath[0]] = value
-        elif kwpath[0] not in params.keys():
+        elif kwpath[0] not in list(params.keys()):
```
Comment (dev-zero) [inline aiida_cp2k/calculations/__init__.py]: `... in ...` works with views, again, no need to wrap in `list(...)`

File: aiida_cp2k/parsers/__init__.py [hunk 5]
```diff
@@ -72,35 +70,35 @@ def _parse_stdout(self, out_folder, new_nodes_list):
...
+        coord_lines = [line.strip().split() for line in match.group(1).split("\n")]
         symbols = [line[0] for line in coord_lines]
         positions_str = [line[1:] for line in coord_lines]
         positions = np.array(positions_str, np.float64)
 
         # parse cell section
-        m = re.search(r'\n\s*&CELL\n(.*?)\n\s*&END CELL\n', content, re.DOTALL)
-        cell_lines = [line.strip().split() for line in m.group(1).split("\n")]
+        match = re.search(r'\n\s*&CELL\n(.*?)\n\s*&END CELL\n', content, re.DOTALL)
+        cell_lines = [line.strip().split() for line in match.group(1).split("\n")]
```
Comment (dev-zero) [inline aiida_cp2k/parsers/__init__.py]: use `.splitlines(...)` instead of `.split("\n")` for more portability

**Pull request merged**